test/cppsuite/tests/test_live_restore.cpp

```diff
@@ -252,7 +252,11 @@ do_random_crud(scoped_session &session, bool fresh_start)
             i = warmup_insertions;
         }
 
-        if (ran < 5000) {
+        if (ran < 3) {
+            // 0.01% Checkpoint.
```

Comment:
```suggestion
            // 0.02% Checkpoint.
```

Reply:
This is actually still 0.01%. It will only trigger if `ran` is `2`